odoo/tools/mail.py

```diff
@@ -22,6 +22,27 @@
 
 _logger = logging.getLogger(__name__)
 
+#----------------------------------------------------------
+# Domains
+#----------------------------------------------------------
+
+# The top 100 email providers
+common_domains = {'gmail.com', 'yahoo.com', 'hotmail.com', 'aol.com', 'hotmail.co.uk', 'hotmail.fr', 'msn.com',
```

Comment:
Should be all-uppercase, and probably starting with an `_` (exactly like the original, basically).

Also not sure this should be *600 lines* away from the one function which uses them.